test/unit/org/apache/cassandra/db/commitlog/CommitLogTest.java

```diff
@@ -209,6 +209,8 @@ public void beforeTest() throws IOException
     @After
     public void afterTest()
     {
+        System.clearProperty("cassandra.replayList");
+        System.clearProperty("cassandra.custom_replay_filter_class");
```

Comment:
`custom_replay_filter_class` seems to be used in cndb but not in CC - can it be removed here?

Reply:
Oh, nvm, I see now that it comes from #374

Reply:
Good that you noticed. It should not be part of this PR :) I will remove it.